tools/start_server.py

```diff
@@ -37,8 +37,16 @@
 import requests
 import zipfile
 
+
+import ffmpeg
+import threading
+import pexpect.fdpexpect
+
+
 WEBSOCKET_PORT = 8081
 FILESERVER_PORT = 8082
+HTML5_STREAMING_HOST = "http://localhost"
+HTML5_STREAMING_PORT = 8085
```

Comment:
Is it important to point exactly HTML**5** in variable names?
I think you can remove `HTML5_` prefix in these names.

Reply:
Not agree. this streaming is specially for sending it to <video> tag ( HTML5)

Reply:
Okek then.